nntrainer/layers/rnn.cpp

```diff
@@ -158,6 +189,19 @@ void RNNLayer::forwarding(bool training) {
     if (!training)
       h_prev.getBatchSlice(b, 1).copy(hs);
   }
+
+  if (!return_sequences) {
+    TensorDim d = hidden_.getDim();
+    for (unsigned int b = 0; b < input_dim[0].batch(); ++b) {
+      float *data = hidden_.getAddress(b * d.width() * d.height() +
+                                       (d.height() - 1) * d.width());
+      float *rdata = net_hidden[0]->getVariableRef().getAddress(b * d.width());
+      std::copy(data, data + d.width(), rdata);
```

Comment:
please try and use `Tensor::copy()`

Reply:
This isn't tensor copy.

Reply:
consider creating tensor wrappers of data, and rdata and then using tensor::copy.
it is not recommended to use std::copy, but rather use Tensor::copy for memory inside tensor.

Reply:
I will change it later pr..

Reply:
alright.